include/ccf/node/quote.h

```diff
@@ -35,10 +38,16 @@ namespace ccf
 
     static std::optional<HostData> get_host_data(const QuoteInfo& quote_info);
 
+    static std::optional<pal::snp::Attestation> get_snp_attestation(
```

Comment:
New public APIs are often worth announcing in the changelog, this one will be of interest for aDNS (@ad-l)